browser-build.config.js

```diff
@@ -1,4 +1,4 @@
-const webpack = require("webpack");
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
```

Comment:
We're running on webpack 4 here. Is the plugin supported?

Reply:
> This module requires a minimum of Node v6.9.0 and Webpack v4.0.0.

The question is rather, why does it work with the other repos?